Lib/cProfile.py

```diff
@@ -152,6 +152,11 @@ def main():
     (options, args) = parser.parse_args()
     sys.argv[:] = args
 
+    # the script that we're profiling may chdir, so capture the absolute path
+    # to the output file at startup
```

Comment:
As per pep8, please start comments with a capital letter and end with a period.
```suggestion
    # The script that we're profiling may chdir, so capture the absolute path
    # to the output file at startup.
```